### ISSUE TITLE
Download historical fundemental data

### ISSUE THREAD

**Issue by NachuanLi** [JECSand/yahoofinancials#85]
Hi, is there a way to download historical fundemental data of stocks such as PE ratio and PB ratio quarterly. I notice that the method provided in yfinance could only download the fundemental datas currently. Hope some one would solve this issue

**Comment by apavlo89** [comment]
Yes, I would like the answer to this as well. How do you get historical data for other data other than open,high,low,close,volume, e.g., for a period of five years?

**Comment by ianverbeck** [comment]
Wondering this as well– historical fundamental data seems hard to find with free APIs.

**Comment by antoinevulcain** [comment]
You can get them from [https://financialmodelingprep.com/developer/docs/](blob:null/aa4db5f0-f7c0-417d-96a5-b70607ddf6f0)
